example/deployments/cumulus/cumulus-sit.tfvars

```diff
@@ -2,4 +2,4 @@ prefix = "cumulus-sit"
 archive_api_port = 8000
 
 include_orca = true
-drop_database = "False"
+drop_database = "True"
```

Comment:
Could we consider updating this/these to be orca_drop_database?

Reply:
We should 😟  😂 We can do it for next orca release, they probably have big changes there.